packages/pluginsdk-peerdeps/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@spinnaker/pluginsdk-peerdeps",
+  "description": "Provides package dependencies to plugin developers",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "scripts": {
+    "usage": "cat USAGE.txt",
+    "stage": "npm run clean && ./sync-stage.sh",
+    "sync": "./sync-save.js",
+    "clean": "./clean.sh"
+  },
+  "files": [],
+  "peerDependencies": {
+    "@spinnaker/core": "0.0.507",
+    "@spinnaker/pluginsdk": "0.0.21",
+    "@uirouter/core": "6.0.4",
+    "@uirouter/react": "1.0.2",
+    "lodash-es": "4.17.15",
+    "prop-types": "15.6.1",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "rxjs": "5.4.2",
+    "@rollup/plugin-commonjs": "15.1.0",
+    "@rollup/plugin-node-resolve": "9.0.0",
+    "@rollup/plugin-typescript": "6.0.0",
```

Comment:
These are the packages+versions of both regular dependencies and devDependencies, all mixed together.